### DISCUSSION TITLE
WHY CAN'T I SETUP CLASSIC XP UP ON WIN 10?

### DISCUSSION THREAD

**Discussion by magnogreato** [community/community#77725] [New to GitHub]
I am a NEWBIE and this site is about 100 ft. above my head...
For some reason when I installed OPEN SHELL and tried to open OPEN SHELL MENU SETTINGS, two things happen...one, MS Edge pops up with AI info even tho I've got FireFox as my default and the other is it assumes I'm into win 11 when in fact I am using Win 10 and I can't get any further with setting up my desktop as I have in the past.  This is a new Win 10 install and I just can't figure out where I went wrong or what's going on to make this so difficult...I didn't have the problem last year, so what's going on?    The Classic View is great and I enjoy it but so far no luck.  Any help/assistance would be greatly appreciated!!!

**Comment by queenofcorgis** [comment]
If you're asking how to use the piece of software like Open Shell that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.
